build-scripts/use-mirror/rhcossync.sh

```diff
@@ -64,6 +67,10 @@ function downloadImages() {
     for img in $(<${SYNCLIST}); do
 	curl -L --retry 5 -O $img
```

Comment:
You might want to add a `--fail` or it will eventually pull a Justin.

Reply:
done, thanks!